ConfirmAccount/ConfirmAccount.config.php

```diff
@@ -1,118 +1,120 @@
-# Each type has its own request queue at Special:ConfirmAccount/<subpage param>.
-# When a request of a certain type is approved, the new user:
-# (a) is placed in the <user group> group (if not User or *)
-# (b) has <autotext> appended to his or her user page
-$wgAccountRequestTypes = array(
-	0 => array( 'authors', 'user', null )
-);
-
-# If set, will add {{DEFAULTSORT:sortkey}} to userpages for auto-categories.
-# The sortkey will be made by doing a regex search and replace on the title.
-# Set this variable to false to avoid sortkey use.
-$wgConfirmAccountSortkey = false;
-// For example, the below will do {{DEFAULTSORT:firstname, lastname}}
-# $wgConfirmAccountSortkey = array( '/^(.+) ([^ ]+)$/', '$2, $1' );
-
-# IMPORTANT: do we store the user's notes and credentials
-# for sucessful account request? This will be stored indefinetely
-# and will be accessible to users with crediential lookup permissions
-$wgConfirmAccountSaveInfo = true;
-
-# Send an email to this address when account requestors confirm their email.
-# Set to false to skip this
-$wgConfirmAccountContact = false;
-
-# If ConfirmEdit is installed and set to trigger for createaccount,
-# inject catpchas for requests too?
-$wgConfirmAccountCaptchas = true;
-
-# Storage repos. Has B/C for when this used FileStore.
-$wgConfirmAccountFSRepos = array(
-	'accountreqs' => array( # Location of attached files for pending requests
-		'name'       => 'accountreqs',
-		'directory'  => isset( $wgFileStore['accountreqs'] ) ?
-			$wgFileStore['accountreqs']['directory'] : "{$IP}/images/accountreqs",
-		'url'        => isset( $wgFileStore['accountreqs'] ) ?
-			$wgFileStore['accountreqs']['url'] : null,
-		'hashLevels' => isset( $wgFileStore['accountreqs'] ) ?
-			$wgFileStore['accountreqs']['hash'] : 3
-	),
-	'accountcreds' => array( # Location of credential files
-		'name'       => 'accountcreds',
-		'directory'  => isset( $wgFileStore['accountcreds'] ) ?
-			$wgFileStore['accountcreds']['directory'] : "{$IP}/images/accountcreds",
-		'url'        => isset( $wgFileStore['accountcreds'] ) ?
-			$wgFileStore['accountcreds']['url'] : null,
-		'hashLevels' => isset( $wgFileStore['accountcreds'] ) ?
-			$wgFileStore['accountcreds']['hash'] : 3
-	)
-);
-
-# Restrict account creation
-$wgGroupPermissions['*']['createaccount'] = false;
-$wgGroupPermissions['user']['createaccount'] = false;
-# Grant account queue rights
-$wgGroupPermissions['bureaucrat']['confirmaccount'] = true;
-# This right has the request IP show when confirming accounts
-$wgGroupPermissions['bureaucrat']['requestips'] = true;
-
-# If credentials are stored, this right lets users look them up
-$wgGroupPermissions['bureaucrat']['lookupcredentials'] = true;
-
-# Show notice for open requests to admins?
-# This is cached, but still can be expensive on sites with thousands of requests.
-$wgConfirmAccountNotice = true;
-
-# End of configuration variables.
-# ########
+<?php
```

Comment:
It just looks like I changed the whole file because I changed the line break mode from \r\n to just \n